### DISCUSSION TITLE
Add RelationType HasOutput, IsOutputOf

### DISCUSSION THREAD

**Discussion by stephw-tib** [datacite/datacite-suggestions#138] [Metadata Schema Suggestions]
### What is the problem that your suggestion solves?

We want to describe the relation between an academic event for which we are minting DOIs and its output - such as proceedings, slides, abstracts, posters, journal articles, etc.. The current options do not fully cover the use case (e.g. one poster is not documenting or describing a whole conference).

### What solution might meet your needs?

adding HasOutput and IsOutputOf to controlled list of values

### Your name

Stephanie Hagemann-Wilholt

### Your organization

Technische Informationsbibliothek (https://ror.org/04aj4c181)

### What alternatives have you tried or considered?

Using values like IsDescribedBy, IsDocumentedBy which is not precise enough for the event-output relation.

### What group(s) would benefit from your suggestion?

- [X] DataCite members registering DOIs and metadata and administering accounts
- [ ] Developers building integrations with DataCite services
- [X] Harvesters who index DataCite DOIs and metadata
- [X] Individuals or institutions analyzing DataCite DOIs and metadata
- [ ] DataCite staff
- [ ] Other

**Comment by rebeccaringuette** [comment]
Adding HasOutput and IsOutputOf to controlled list of values also provides a link between software and the data it produces. This is extremely important for transparency and reproducibility to indicate for computational models and simulations so the data those software produce are directly connected to the version-specific DOI of the software that produced them.

**Comment by KellyStathis** [comment]
Hi @stephw-tib, thank you for sharing this suggestion! The Metadata Working Group has been discussing this use case and the suggested relationType pair "HasOutput" and "IsOutputOf". However, we're not finding it intuitive to refer to presentations, posters, and other materials as "outputs" of academic events, such as conferences.

We think that "IsPartOf" and "HasPart" could be suitable for this use case. For example, the metadata for a presentation could have this RelatedIdentifier to link to the academic event (where in this example, the event has DOI `10.21384/abcd`):

```xml
<relatedIdentifier relatedIdentifierType="DOI" relationType="IsPartOf" resourceTypeGeneral="Event">10.21384/abcd</relatedIdentifier>
```

We welcome any feedback on this alternative, and we are interested in continuing to explore this further with you.
